modules/hardware-qualcomm.nix

```diff
@@ -52,6 +57,12 @@ in
         quirks.qualcomm.msm-fb-refresher.enable = true;
       };
     }
+    {
+      mobile = mkIf cfg.qualcomm-msm8974.enable {
+        system.system = "armv7a-linux";
+        quirks.qualcomm.msm-fb-handle.enable = true;
+      };
+    }
```

Comment:
Though Android themselves use `armv7a` as the system type of their native 32 bit ARM builds, Nixpkgs and NixOS use `armv7l` for the generic ARMv7 platform.

Using `armv7a` would build, but you will lose the ability to use the upcoming armv7l binary cache.

If it doesn't work with `armv7l-linux`, we'll (I'll) have to investigate, I believe it should.

Reply:
Ok. Let's try with `armv7l` with the newly pushed commit.